PyU4V/tools/openstack/migrate.py

```diff
@@ -0,0 +1,192 @@
+                    element_dict,
+                    masking_view_components['portgroup'],
+                    masking_view_components['initiatorgroup'],
+                    is_revert)
+        else:
+            utils.smart_print(
+                'NO MIGRATION', migrate_utils.WARNING)
+            sys.exit()
+    return masking_view_components, element_dict, no_action
+
+
+def migrate_case(masking_view_name):
+    """The revert case of the migrate process
+
+    :param masking_view_name: masking view name -- str
+    :returns: masking view details -- dict
+              element details -- dict
+              no action flag -- boolean
+    """
+    masking_view_components, element_dict = {}, {}
+    no_action = True
+    if utils.check_masking_view_for_migration(masking_view_name):
+        no_action = False
+        utils.smart_print(
+            'OLD MASKING VIEW IS %s',
+            migrate_utils.DEBUG, masking_view_name)
+        masking_view_components = (
+            utils.get_elements_from_masking_view(masking_view_name))
+        # Compile the new names of the SGs and MV
+        element_dict = utils.compile_new_element_names(
+            masking_view_name, masking_view_components['portgroup'],
+            masking_view_components['initiatorgroup'],
+            masking_view_components['storagegroup'])
+        # Check if masking view exists and if it does validate it
+        utils.get_or_create_masking_view(
+            element_dict, masking_view_components['portgroup'],
+            masking_view_components['initiatorgroup'])
+    return masking_view_components, element_dict, no_action
+
+
+def move_volumes(masking_view_components, element_dict):
+    """Move volumes from one masking view to another
+
+    :param masking_view_components: masking view details -- dict
+    :param element_dict: element details -- dict
+    """
+    # Check the qos setting of source and target storage group
+    utils.set_qos(
+        masking_view_components['storagegroup'],
+        element_dict['new_sg_name'])
+    volume_list, create_volume_flag = utils.get_volume_list(
+        masking_view_components['storagegroup'])
+    if volume_list:
+        message = utils.move_volumes_from_source_to_target(
+            volume_list, masking_view_components['storagegroup'],
+            element_dict['new_sg_name'], create_volume_flag)
+        print_str = '%s SOURCE STORAGE GROUP REMAINS'
+        utils.smart_print(
+            print_str, migrate_utils.DEBUG,
+            masking_view_components['storagegroup'])
+        utils.print_pretty_table(message)
+        new_storage_group = utils.get_storage_group(
+            element_dict['new_sg_name'])
+        print_str = '%s TARGET STORAGE GROUP DETAILS:'
+        utils.smart_print(
+            print_str, migrate_utils.DEBUG,
+            element_dict['new_sg_name'])
+        utils.print_pretty_table(new_storage_group)
+
+for masking_view in masking_view_list:
```

Comment:
Two lines between methods outside of classes, getting a pep8 warning in PyCharm but not in the pep8 tox tun